utils/swift-xcodegen/Tests/SwiftXcodeGenTest/NinjaParserTests.swift

```diff
@@ -2,52 +2,63 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Copyright (c) 2024 - 2025 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
 
-import XCTest
+import Foundation
+import Testing
+
 @testable import SwiftXcodeGen
 
 fileprivate func expectEqual<T: Equatable>(
```

Comment:
I only added this because the XCTest matching for arrays doesn't give a helpful message on failure, I assume swift-testing should be better? i.e I think calls to this could be replaced by `#expect(xs == ys)`